continue-reading.html

```diff
@@ -1,13 +1,231 @@
 <!DOCTYPE HTML>
 <html>
-  <head>
-    <title>Continue Reading</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+
+<head>
+  <title>Motherflippin Homepage</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <link href="style.css" rel="stylesheet" />
+</head>
+
+
+<body id="container">
+
+
+
+
+  <!-- header section -->
+  <div class="item header">

```

Comment:
using `class="item header"` is giving two classes to this `<div>`. In your CSS you don't have any class named `item`